src/reducers/application.js

```diff
@@ -44,7 +44,9 @@ export default (state = initialState, action) => {
     case MENU_LOADED: {
       return {
         ...state,
-        menuLinks: action.payload.menuLinks,
+        menuLinks: Array.isArray(action.payload.menuLinks)
```

Comment:
How about falling back to state.menuLinks?

Reply:
`action.payload.menuLinks` can we just check the length?

Reply:
Things which aren't arrays can also have a .length property